changelog.md

```diff
@@ -1,6 +1,11 @@
+5.0.0
+========
+- Introduce a unique identifier for a CredenentialDescription object. See [PR](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/94) for details.
+- Change `AuthorizationHeaderProviderOptions` to use a `string` instead of `HttpMethod`. See [PR](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/99) for details.

```

Comment:
```suggestion
- Change `AuthorizationHeaderProviderOptions` to use a `string` instead of `HttpMethod`. See [PR](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/99) for details. This is a breaking change, but shouldn't affect you if you are using the configuration.

```